### PR TITLE
[Intel MKL] Update TF to fuse Conv2d+Relu6/Elu

### PR DESCRIPTION
**Pull request by guizili0**
We first disable Conv2d+Relu6/Elu in MKL build as lack of the backend support, now add the support in backend and remove the limitation in Conv2d fusion.

### CODE REVIEW

File: tensorflow/core/kernels/mkl_fused_ops_test.cc [hunk 1]
```diff
@@ -310,6 +321,30 @@ TYPED_TEST_P(MklFusedConv2DWithBiasOpTest, SpatialConvolutionAndRelu) {
   this->VerifyFusedConv2D(filter_size, filter_count, {"BiasAdd", "Relu"});
 }
 
+TYPED_TEST_P(MklFusedConv2DWithBiasOpTest, OneByOneConvolutionAndRelu6) {
+  const int filter_size = 1;
```
Comment (penpornk) [inline tensorflow/core/kernels/mkl_fused_ops_test.cc]: Per our [Style Guide](https://google.github.io/styleguide/cppguide.html#Constant_Names) on naming constants,
```suggestion
  const int kFilterSize = 1;
```
Please change other places as well.

**Review by penpornk — CHANGES_REQUESTED** [review]
Thank you for your PR! I have minor formatting comments.

**Comment by guizili0** [comment]
@penpornk thanks for the comments, I have update my code based on your comments. Please help to check, thanks.

File: tensorflow/core/kernels/mkl_fused_ops_test.cc [hunk 2]
```diff
@@ -201,7 +201,7 @@ class MklFusedConv2DOpTest : public OpsTestBase {
     if (std::find(fused_ops.begin(), fused_ops.end(), "Relu6") !=
         fused_ops.end()) {
       last_op = "with_relu6";
-      next_op = ops::Relu(root.WithOpName(last_op), next_op);
+      next_op = ops::Relu6(root.WithOpName(last_op), next_op);
```
Comment (penpornk) [inline tensorflow/core/kernels/mkl_fused_ops_test.cc]: Nice catch!

**Review by penpornk — APPROVED** [review]
Thank you for the prompt fixes!

**Pull request merged**